// The Firmament project
// Copyright (c) 2011-2012 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// Task-related messages, convering task creation and task completion.

syntax = "proto3";

package firmament;

import "base/task_perf_statistics_sample.proto";

message TaskHeartbeatMessage {
  uint64 task_id = 1;
  string location = 2;
  uint64 sequence_number = 3;
  TaskPerfStatisticsSample stats = 4;
}
